### PR TITLE
IfcCableFitting Rail Import BIMQ 2021-07-21T06:32:04.5156286Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcCableFitting

### CODE REVIEW

File: IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj" Name="Attenuation" UniqueId="0fc49883-5ab2-40aa-b640-e276e1d814ed" PrimaryDataType="IfcSoundPowerMeasure" />
```
Comment (jmirtsch) [inline IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/DocProperty.xml:2]:
```suggestion
<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj" Name="Attenuation" UniqueId="0fc49883-5ab2-40aa-b640-e276e1d814ed" PrimaryDataType="IfcSoundPowerLevelMeasure" />
```

Comment (jmirtsch) [reply IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/DocProperty.xml:2]: IfcSoundPowerMeasure was renamed in IFC4 https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/HTML/link/ifcsoundpowerlevelmeasure.htm

File: IFC4x3/Properties/e/EndPointId_3ctuSusCj2_w79HgKqMiDN/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="EndPointId_3ctuSusCj2_w79HgKqMiDN" Name="EndPointId" UniqueId="e6df8738-d8cb-42fb-a1c9-46a5345ac357" PrimaryDataType="IfcIdentifier" />
```
Comment (jmirtsch) [inline IFC4x3/Properties/e/EndPointId_3ctuSusCj2_w79HgKqMiDN/DocProperty.xml:2]: Is this a property? Or a relationship such as port connectivity.  It shouldn't be in a type driven override pset as I understand it.

File: IFC4x3/Properties/s/StartPointId_0OLNfz0Ir4oRaSoNkqZHEH/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Indicate the ending point of the cable
```
Comment (jmirtsch) [inline IFC4x3/Properties/s/StartPointId_0OLNfz0Ir4oRaSoNkqZHEH/Documentation.md:1]:
```suggestion
Indicate the starting point of the cable
```

File: IFC4x3/Properties/s/StartPointId_0OLNfz0Ir4oRaSoNkqZHEH/DocProperty.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="StartPointId_0OLNfz0Ir4oRaSoNkqZHEH" Name="StartPointId" UniqueId="18557a7d-012d-44c9-b91c-c97bb48d1391" PrimaryDataType="IfcIdentifier" />
```
Comment (jmirtsch) [inline IFC4x3/Properties/s/StartPointId_0OLNfz0Ir4oRaSoNkqZHEH/DocProperty.xml:2]: Is this a property? Or a relationship such as port connectivity.  It shouldn't be in a type driven override pset as I understand it.

File: IFC4x3/Properties/e/EndPointId_3ctuSusCj2_w79HgKqMiDN/Documentation.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Indicate the starting point of the cable
```
Comment (jmirtsch) [inline IFC4x3/Properties/e/EndPointId_3ctuSusCj2_w79HgKqMiDN/Documentation.md:1]:
```suggestion
Indicate the ending point of the cable
```

File: IFC4x3/Properties/g/GroundResistance_2NdoXdKT98XeEMmk8dSBQF/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+The resisrance through earthing terminal exit to the ground.
```
Comment (jmirtsch) [inline IFC4x3/Properties/g/GroundResistance_2NdoXdKT98XeEMmk8dSBQF/Documentation.md:1]:
```suggestion
The resistance through earthing terminal exit to the ground.
```

File: IFC4x3/Properties/o/OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk/Documentation.md [hunk 7]
```diff
@@ -0,0 +1 @@
+Outer diameter of the cable(mm)
```
Comment (jmirtsch) [inline IFC4x3/Properties/o/OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk/Documentation.md:1]:
```suggestion
Outer diameter of the cable
```

File: IFC4x3/Properties/o/OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk" Name="OverallDiameter" UniqueId="c5203793-1a63-4698-b798-6531a31eb6ee" PrimaryDataType="IfcPositiveLengthMeasure" />
```
Comment (jmirtsch) [inline IFC4x3/Properties/o/OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml:2]: Should this use common Nominal Diameter Property instead?

File: IFC4x3/PropertyConstants/s/STANDARTSINGLEMODEFIBER_0RhS8$7ZTFqftLR1K2iYTe.xml [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="STANDARTSINGLEMODEFIBER_0RhS87ZTFqftLR1K2iYTe" Name="STANDARTSINGLEMODEFIBER" UniqueId="1badc23f-1e37-4fd2-9dd5-6c1502b22768" />
```
Comment (jmirtsch) [inline IFC4x3/PropertyConstants/s/STANDARTSINGLEMODEFIBER_0RhS8$7ZTFqftLR1K2iYTe.xml:2]: Should this be renamed to STANDARDSINGLEMODEFIBER ?

File: IFC4x3/Properties/s/StartPointId_0OLNfz0Ir4oRaSoNkqZHEH/DocProperty.xml (hunk 4 above)
Comment (fhulin) [reply IFC4x3/Properties/s/StartPointId_0OLNfz0Ir4oRaSoNkqZHEH/DocProperty.xml:2]: It's an unclear point for the Rail Telecom Domain. @czapplitec

File: IFC4x3/Properties/e/EndPointId_3ctuSusCj2_w79HgKqMiDN/DocProperty.xml (hunk 2 above)
Comment (fhulin) [reply IFC4x3/Properties/e/EndPointId_3ctuSusCj2_w79HgKqMiDN/DocProperty.xml]: Backlog. Commit : aef1a4f5c5a6cd1f4c4624d1f3a06b988d1c1586

File: IFC4x3/Properties/o/OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml (hunk 8 above)
Comment (fhulin) [reply IFC4x3/Properties/o/OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml]: Create generic NominalDiameter  85978b727ae68689aab5f9a94269ca18fefaf981
Need to discuss Friday!

File: IFC4x3/Properties/s/StartPointId_0OLNfz0Ir4oRaSoNkqZHEH/DocProperty.xml (hunk 4 above)
Comment (fhulin) [reply IFC4x3/Properties/s/StartPointId_0OLNfz0Ir4oRaSoNkqZHEH/DocProperty.xml]: Backlog. Commit : aef1a4f5c5a6cd1f4c4624d1f3a06b988d1c1586

File: IFC4x3/Properties/o/OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml (hunk 8 above)
Comment (jmirtsch) [reply IFC4x3/Properties/o/OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml]: If possible, we should harmonize the other relevant properties with a generic description.  Typically we picked the first one and changed the description.

File: IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/Documentation.md [hunk 10]
```diff
@@ -0,0 +1 @@
+Indicate the Optical or Electrical Attenuation of the cable measured in dB, at a certain wavelength or frequency, always changing with the length of the cable.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/Documentation.md]:
```suggestion
Indicates the optical or electrical attenuation of the cable measured in dB, at a certain wavelength or frequency, always changing with the length of the cable.
```

File: IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/DocProperty.xml [hunk 11]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj" Name="Attenuation" UniqueId="0fc49883-5ab2-40aa-b640-e276e1d814ed" PrimaryDataType="IfcSoundPowerLevelMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/DocProperty.xml]: We should look into existing noise emissions properties.

File: IFC4x3/Properties/c/CableArmourType_3$2Up8Fxb2RPdKNA2Tyn0M/Documentation.md [hunk 12]
```diff
@@ -0,0 +1 @@
+The armour type of the cable for mechanical protection
```
Comment (SergejMuhic) [inline IFC4x3/Properties/c/CableArmourType_3$2Up8Fxb2RPdKNA2Tyn0M/Documentation.md]:
```suggestion
The armour type of the cable for mechanical protection.
```

File: IFC4x3/Properties/c/CableFunction_185zEg3BX9UPnOrdfFpNFk/Documentation.md [hunk 13]
```diff
@@ -0,0 +1 @@
+To distinguish between Telecom and Power Supply cables
```
Comment (SergejMuhic) [inline IFC4x3/Properties/c/CableFunction_185zEg3BX9UPnOrdfFpNFk/Documentation.md]:
```suggestion
Distinguishes between Telecom and Power Supply cables.
```

File: IFC4x3/Properties/c/ConnectorAType_05lh3wrjD3fe$BEp5kTIqH/Documentation.md [hunk 14]
```diff
@@ -0,0 +1 @@
+Indicate the type of A-end connector
```
Comment (SergejMuhic) [inline IFC4x3/Properties/c/ConnectorAType_05lh3wrjD3fe$BEp5kTIqH/Documentation.md]:
```suggestion
Indicates the type of A-end connector.
```

File: IFC4x3/Properties/c/ConnectorBType_2xjdteWAjCdRxhbBMzx0M0/Documentation.md [hunk 15]
```diff
@@ -0,0 +1 @@
+Indicate the type of B-end connector
```
Comment (SergejMuhic) [inline IFC4x3/Properties/c/ConnectorBType_2xjdteWAjCdRxhbBMzx0M0/Documentation.md]:
```suggestion
Indicates the type of B-end connector.
```

File: IFC4x3/Properties/f/FiberType_33hW6SI9L88u4r9StkBaiz/Documentation.md [hunk 16]
```diff
@@ -0,0 +1 @@
+Indicate the type of the single fiber
```
Comment (SergejMuhic) [inline IFC4x3/Properties/f/FiberType_33hW6SI9L88u4r9StkBaiz/Documentation.md]:
```suggestion
Indicates the type of the single fiber.
```

File: IFC4x3/Properties/f/FireRetardantClass_34JNOUumbFZ96jULEv9Mf0/Documentation.md [hunk 17]
```diff
@@ -0,0 +1 @@
+Indicate fire retardant class of the cable.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/f/FireRetardantClass_34JNOUumbFZ96jULEv9Mf0/Documentation.md]:
```suggestion
Indicates fire retardant class of the cable.
```

File: IFC4x3/Properties/f/FireRetardantClass_34JNOUumbFZ96jULEv9Mf0/DocProperty.xml [hunk 18]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FireRetardantClass_34JNOUumbFZ96jULEv9Mf0" Name="FireRetardantClass" UniqueId="c44d761e-e309-4f8c-91ad-7953b9256a40" PrimaryDataType="IfcLabel" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/f/FireRetardantClass_34JNOUumbFZ96jULEv9Mf0/DocProperty.xml]: Could probably modify and reuse `FireRating`?

File: IFC4x3/Properties/f/FunctionType_0vO1lA0UzAdubt362mA3py/Documentation.md [hunk 19]
```diff
@@ -0,0 +1 @@
+Indicate whether the connector is a Telecom connector or a Power Supply connector.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/f/FunctionType_0vO1lA0UzAdubt362mA3py/Documentation.md]:
```suggestion
Indicates whether the connector is a Telecom connector or a Power Supply connector.
```

File: IFC4x3/Properties/f/FunctionType_0vO1lA0UzAdubt362mA3py/DocProperty.xml [hunk 20]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FunctionType_0vO1lA0UzAdubt362mA3py" Name="FunctionType" UniqueId="39601bca-01ef-4a9f-8977-0c60b0283cfc" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```
Comment (SergejMuhic) [inline IFC4x3/Properties/f/FunctionType_0vO1lA0UzAdubt362mA3py/DocProperty.xml]: A bit ambiguous. Could the naming be made clearer?

File: IFC4x3/Properties/g/GroundResistance_2NdoXdKT98XeEMmk8dSBQF/Documentation.md [hunk 21]
```diff
@@ -0,0 +1 @@
+The resistance through earthing terminal exit to the ground.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/g/GroundResistance_2NdoXdKT98XeEMmk8dSBQF/Documentation.md]: This should be rephrased. It is not clear.

File: IFC4x3/Properties/i/IsFireResistant_0EDQmMlDr4wOieHoEptGkG/Documentation.md [hunk 22]
```diff
@@ -0,0 +1 @@
+Whether or not the cable is fire resistant
```
Comment (SergejMuhic) [inline IFC4x3/Properties/i/IsFireResistant_0EDQmMlDr4wOieHoEptGkG/Documentation.md]:
```suggestion
Informs whether the cable is fire resistant.
```

File: IFC4x3/Properties/n/NominalDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml [hunk 23]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NominalDiameter_3583UJ6cD6cBUOPJ6Z7hRk" Name="NominalDiameter" UniqueId="c5203793-1a63-4698-b798-6531a31eb6ee" PrimaryDataType="IfcPositiveLengthMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NominalDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml:2]: Reuse.

File: IFC4x3/Properties/o/OutJacketColour_1vHztIiLn2ZvG$H5JLPMVP/Documentation.md [hunk 24]
```diff
@@ -0,0 +1 @@
+Indicate the color of the outer jacket of the cable
```
Comment (SergejMuhic) [inline IFC4x3/Properties/o/OutJacketColour_1vHztIiLn2ZvG$H5JLPMVP/Documentation.md]:
```suggestion
Indicates the color of the outer jacket of the cable.
```

File: IFC4x3/Properties/o/OutJacketColour_1vHztIiLn2ZvG$H5JLPMVP/DocProperty.xml [hunk 25]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="OutJacketColour_1vHztIiLn2ZvGH5JLPMVP" Name="OutJacketColour" UniqueId="7947ddd2-b15c-428f-943f-4454d56567d9" PrimaryDataType="IfcLabel" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/o/OutJacketColour_1vHztIiLn2ZvG$H5JLPMVP/DocProperty.xml]: Rename `OuterJacketColour` or just `JacketColour`?

File: IFC4x3/Properties/t/TubeDiameter_2$VL9j18b63QdizLrkdfFs/DocProperty.xml [hunk 26]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TubeDiameter_2VL9j18b63QdizLrkdfFs" Name="TubeDiameter" UniqueId="bf7d526d-0489-460d-a9ec-f55d6e9e93f6" PrimaryDataType="IfcPositiveLengthMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/TubeDiameter_2$VL9j18b63QdizLrkdfFs/DocProperty.xml:2]: Reuses `NominalDiameter`?

File: IFC4x3/PropertyConstants/p/POWERSUPPLY_1Tm2DC7AD9of9tt5MLICq2.xml [hunk 27]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="POWERSUPPLY_1Tm2DC7AD9of9tt5MLICq2" Name="POWERSUPPLY" UniqueId="5dc0234c-1ca3-49ca-9277-dc559548cd02" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/p/POWERSUPPLY_1Tm2DC7AD9of9tt5MLICq2.xml]: Seems like a duplicate.

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CableFittingTypeExit/Documentation.md [hunk 28]
```diff
@@ -0,0 +1 @@
+Properties of the exit type of cable fitting, which ends a cable segment at a non-electric element.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CableFittingTypeExit/Documentation.md]:
```suggestion
Properties of the exit type of cable fitting which ends a cable segment at a non-electric element.
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_OpticalAdapter/Documentation.md [hunk 29]
```diff
@@ -0,0 +1 @@
+Properties in this property set are applicable to the transition type of cable fitting, indicated that such transtion is an optical adapter.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_OpticalAdapter/Documentation.md]: Something is wrong with the second sentence.
```suggestion
Properties in this property set are applicable to the transition type of cable fitting. Indicated that such transition is an optical adapter.
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_TelecomCableGeneral/DocPropertySet.xml [hunk 30]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_TelecomCableGeneral/DocPropertySet.xml:1]: Documentation missing.

**Review by SergejMuhic — COMMENTED** [review]
Property constant documentation. Consistent use of underscores.
PEnum documentation.

File: IFC4x3/PropertyConstants/t/TELECOMMUNICATION_2irLJH5$nENOFPVXeJQa7c.xml [hunk 31]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TELECOMMUNICATION_2irLJH5nENOFPVXeJQa7c" Name="TELECOMMUNICATION" UniqueId="acd554d1-17fc-4e5d-83d9-7e1a136a41e6" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/PropertyConstants/t/TELECOMMUNICATION_2irLJH5$nENOFPVXeJQa7c.xml]: Documentation

File: IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/Documentation.md [hunk 32]
```diff
@@ -0,0 +1 @@
+Indicates the optical or electrical attenuation of the cable measured in dB, at a certain wavelength or frequency, always changing with the length of the cable.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/Documentation.md]:
```suggestion
Indicates the optical or electrical attenuation of the cable measured in dB, at a certain wavelength or frequency, changing with the length of the cable.
```

File: IFC4x3/Properties/i/IsFireResistant_0EDQmMlDr4wOieHoEptGkG/Documentation.md [hunk 33]
```diff
@@ -0,0 +1 @@
+Informs whether the cable is fire resistant.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/i/IsFireResistant_0EDQmMlDr4wOieHoEptGkG/Documentation.md]:
```suggestion
Indicates whether the cable is fire resistant.
```

File: IFC4x3/Properties/j/JacketColour_1vHztIiLn2ZvG$H5JLPMVP/Documentation.md [hunk 34]
```diff
@@ -0,0 +1 @@
+Indicates the color of the outer jacket of the cable.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/j/JacketColour_1vHztIiLn2ZvG$H5JLPMVP/Documentation.md]:
```suggestion
Indicates the colour of the outer jacket of the cable or fitting.
```

File: IFC4x3/Properties/g/GroundResistance_2NdoXdKT98XeEMmk8dSBQF/Documentation.md [hunk 35]
```diff
@@ -0,0 +1 @@
+The resistance soil or ground resists electric current from the cable fitting.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/g/GroundResistance_2NdoXdKT98XeEMmk8dSBQF/Documentation.md]:
```suggestion
The soil or ground resistance to electrical current from the cable fitting
```

File: IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/DocProperty.xml (hunk 11 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/DocProperty.xml]: Needs changing to IfcReal

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]

File: IFC4x3/Properties/n/NominalDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml (hunk 23 above)
Comment (fhulin) [reply IFC4x3/Properties/n/NominalDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml:2]: Already discussed in Cable Segment. All the existing one are not generic and have japanese translation, so we have created a new one that we are reusing here.

File: IFC4x3/Properties/t/TubeDiameter_2$VL9j18b63QdizLrkdfFs/DocProperty.xml (hunk 26 above)
Comment (fhulin) [reply IFC4x3/Properties/t/TubeDiameter_2$VL9j18b63QdizLrkdfFs/DocProperty.xml:2]: @czapplitec  do you have comments?

File: IFC4x3/PropertyEnumerations/f/PEnum_FiberType.xml [hunk 36]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_FiberType" Name="PEnum_FiberType" UniqueId="f9167f83-06ac-403e-ad56-97e948f3835d">
```
Comment (fhulin) [inline IFC4x3/PropertyEnumerations/f/PEnum_FiberType.xml:2]: c88812ba98efa9af1e4de31bf345870dedc28a76

File: IFC4x3/PropertyConstants/t/TELECOMMUNICATION_2irLJH5$nENOFPVXeJQa7c.xml (hunk 31 above)
Comment (fhulin) [reply IFC4x3/PropertyConstants/t/TELECOMMUNICATION_2irLJH5$nENOFPVXeJQa7c.xml]: c88812ba98efa9af1e4de31bf345870dedc28a76

File: IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/DocProperty.xml (hunk 11 above)
Comment (fhulin) [reply IFC4x3/Properties/a/Attenuation_0Fn9Y3Mh90ghP0udRXs1Jj/DocProperty.xml]: Done. c88812ba98efa9af1e4de31bf345870dedc28a76

**Review by AlexBrad1eyCT — APPROVED** [review]
Happy with a new nominal diameter will update existing ones accordingly in separate clean up PR, tube diameter is fine its relevent as discussed in meeting. LFTM.
@SergejMuhic please place approval/review the we can merge

File: IFC4x3/Properties/g/GroundResistance_2NdoXdKT98XeEMmk8dSBQF/Documentation.md [hunk 37]
```diff
@@ -0,0 +1 @@
+The soil or ground resistance to electrical current from the cable fitting
```
Comment (SergejMuhic) [inline IFC4x3/Properties/g/GroundResistance_2NdoXdKT98XeEMmk8dSBQF/Documentation.md]: I still do not understand this. The cable fitting induces resistance in the ground/soil?
```suggestion
The soil or ground resistance to electrical current from the cable fitting.
```

File: IFC4x3/Properties/j/JacketColour_1vHztIiLn2ZvG$H5JLPMVP/Documentation.md [hunk 38]
```diff
@@ -0,0 +1 @@
+Indicates the colour of the outer jacket of the cable or fitting.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/j/JacketColour_1vHztIiLn2ZvG$H5JLPMVP/Documentation.md]:
```suggestion
Indicates the colour of the cable or fitting jacket.
```

**Review by SergejMuhic — CHANGES_REQUESTED** [review]
Could we remove the `ChangeSet` changes?

**Review by AlexBrad1eyCT — APPROVED** [review]

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**